Godeps/Godeps.json

```diff
@@ -1171,127 +1171,127 @@
 		},
 		{
 			"ImportPath": "github.com/gogo/protobuf/gogoproto",
-			"Comment": "v0.4-3-gc0656edd",
+			"Comment": "v0.4-3-gc0656ed",
```

Comment:
I agree that this is something that should be addressed however I don't believe this is the scope of this PR as neither this or the previous one changed the commit number.